search-app/src/cmr/search/services/query_service.clj

```diff
@@ -264,6 +267,84 @@
         results (qe/execute-query context query)]
     (common-search/search-results->response context query results)))
 
+(defn- get-new-granules
+  "Finds granules that were added after a given date. Supports CMR Harvesting."
+  [context created-at]
+  (if-let [created-at (time-format/parse created-at)]
+    (let [query (qm/query {:concept-type :granule
+                           :condition (qm/date-range-condition
+                                       :created-at created-at nil)
+                           :page-size :unlimited
+                           :result-format :query-specified
+                           :result-fields [:entry-title :provider-id :collection-concept-id]})
+          new-granules (qe/execute-query context query)]
+     (:items new-granules))
+    nil))
+
+(defn- get-highest-visible-revisions
```

Comment:
Why are we looking at the all collection revisions index? After reviewing more it looks like this was just moved, but want to make sure we aren't using this for this ticket.